features/support/env.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "./../lib/symfony"
+require_relative "./../lib/utils"
+
+PROJECT_ROOT = File.realpath("#{__dir__}/../../")
+FIXTURE_PATH = File.realpath("#{PROJECT_ROOT}/features/fixtures/#{Symfony.fixture}")
+
+# TODO: Maze.hooks.after_configuration doesn't seem to work ?
+AfterConfiguration do
+  # log to console, not the file
+  Maze.config.file_log = false
+  Maze.config.log_requests = true
+
+  # don't wait so long for requests/not to receive requests
+  Maze.config.receive_requests_wait = 10
+  Maze.config.receive_no_requests_wait = 10
+
+  # bugsnag-symfony doesn't send the integrity header (because it's not needed)
+  Maze.config.enforce_bugsnag_integrity = false
+
+  if ENV["DEBUG"]
+    puts "Installing bugsnag-symfony from '#{PROJECT_ROOT}' to '#{FIXTURE_PATH}'"
```

Comment:
Does this come out as you want?  I've come to use `STDOUT.puts`, but I can't remember if it formats it differently or doesn't come out at all with just `puts`.

Reply:
There shouldn't be a difference — `puts` is a shortcut to `$stdout.puts`

https://rubyapi.org/3.0/o/kernel#method-i-puts